\documentclass[9pt,twocolumn,twoside]{gsajnl}

\articletype{inv} % article type
\newcommand{\X}{\textcolor{red}{\bf X\,}}
\newcommand{\citex}{\textcolor{red}{\bf (CITE)\,}}
\newcommand{\jri}[1]{\textcolor{red}{ \emph{ #1}} }
\newcommand{\kc}[1]{\textcolor{blue}{ \emph{ #1}} }

\title{The dynamics of selection and adaptive diversity in modern maize inbred lines}

\author[$\ast$,1]{Kate Crosby}
\author[$\S$]{Justin Gerke}
\author[$\dagger$]{Oscar ``Howie'' Smith}
\author[$\ddagger$,1]{Jeffrey Ross-Ibarra}
%\author[$\ast\ast$]{Author Five}

\affil[$\ast$]{Author one affiliation}
\affil[$\S$]{Author two affiliation}
\affil[$\dagger$]{Author three affiliation}
\affil[$\ddagger$]{Author four affiliation}
%\affil[$\ast\ast$]{Author five affiliation}

\keywords{maize; inbreds; selection; adaptive diversity}

\runningtitle{Modern maize adaptive diversity} % For use in the footer 

\correspondingauthor{Corresponding Author}

\begin{abstract}

blah blah

\end{abstract}

\setboolean{displaycopyright}{true}

\begin{document}

\maketitle
\thispagestyle{firststyle}
\marginmark
\firstpagefootnote
\correspondingauthoraffiliation{Please insert the affiliation correspondence address and email for the corresponding author. The corresponding author should be marked with a `1' in the author list, as shown in the example.}
\vspace{-11pt}%


%For the authors' names, indicate different affiliations with the symbols: $\ast$,  $\S$, $\dagger$, $\ddagger$. After four authors, the symbols double, triple, quadruple, and so forth as required.


\section*{Introduction}

\lettrine[lines=2]{\color{color2}M}{}aize is a staple crop in the United States making up X agriculture acres and producing X weight in animal feed. 
Maize yields (a fitness proxy) in the United States have increased in terms of total numbers since the late 1800s, but have largely slowed in terms of growth rate since the influx of new adaptive material in the mid 1950s to early 1970s. As growth rate in fitness is generally correlated with the amount of available adaptive genetic diversity 
Total decrease in heterozygosity amongst and within heterotic breeding pools (cite Gerke - maybe give more of an example here) of modern maize, suggests that genetic diversity is lower than it once was. 
If much of the diversity that has been lost is adaptive, then present day lines would have less variation available to respond to various selection regimes imposed by breeders. 
If diversity has indeed been lost over time, and has not been recouped by the introgression of new material or mutational input, then what is the composition of this lost genetic diversity? Previous studies suggest that \kc{find lit. on this}...
\jri{note in gerke paper that haplotypes fixed (presumably under selection) not found in lines derived from those cycles, suggesting perhaps that adaptive haps have been lost.}
If the loss is largely composed of deleterious or neutral alleles, then the loss is not of concern to farmers and breeders. 
If this loss contains even a small amount of beneficial/adaptive alleles, this would mean that the efficacy of selection on modern maize lines would suffer as time went on. 
This is because, save for the inputs of novel variation in the 1950s when heterotic breeding pools were originally formed - there has not been much in terms of inputs from novel maize material save for... \kc{cite tropical introgression for blight stuff}.


The "inbred-hybrid" selection method for propagating and distributing maize hybrids has been in use for over 100 years in many breeding platforms and programs in the United States. 
This method selects on inbred "combining ability" (under a given selection program) to produce a vigorous and viable hybrid, which is then grown by farmers. 
The loss of heterosis or "hybrid breakdown" then occurs in the next generation, with a loss in fitness/yield.
Phenotypic selection by breeders using mapping populations was a common practice prior to the advent of molecular markers. 
While past studies have found signatures of selection in modern maize: list (cite,cite,cite) these studies have largely fallen short in finding signatures of soft selection and the polygenic basis of likely many traits under the influence of selection. 
Large effect quantitative trait loci (QTL) and SNPs at these loci are generally markers that are consistently observed to be associated with traits across or within environments. 
However, the genetic basis of most adaptive traits is likely to be polygenic in nature affected by many alleles of small effect \kc{cite examples Doebley, etc Rockman review, Yeaman et al 2015}. We attempt to apply a new method to isolate this signature of polygenic adaptation to our panel of inbred lines

We focus narrowly on an inbred line's year release date to pinpoint these signatures of selection through time (compare and contrast with Joost) within a heterotic group or genetic cluster as past studies have failed to account for population structure, which may influence a study's power to detect selection \kc{explain the concept of heterotic group or breeding pool above?}. 



\section*{Materials and Methods}

In order to examine temporal selection through time we obtained the exact release dates of 823 public maize inbred lines, using a variety of sources including: Gerdes and Tracy, the USDA database, original line release sheets, or communication with maize breeding experts. Many dates in the USDA although categorized as "Year Released" are in fact the year that the line was acquired. 


\subsection*{Statistical Analysis} 
To isolate signals of selection and local adaptation within each genetic cluster of maize inbreds,  we used BayEnv2, which estimates Bayes factors, and $\beta$ broadly approximates the parameter $Ns$ according to a linear model. 
We estimated the neutral covariance matrix of a group of 10,000 GBS markers by choosing every  GBS marker by chromosome. Given that there was approximately  these markers are at the very least not cis-linked, which is key to estimating the covariance matrix. 
The neutral covariance matrix is used to estimate the null model of evolution that can be expected across environments, in this case temporal environments. 
Bayes Factors that deviated or fell outside the n

First we grouped inbred lines according to their broad genetic cluster (or 'heterotic group') using principal components analysis (PCA) using the approach described in 
PCA using flashpca \kc{cite plos one G Abraham}.  
Maize inbred lines within a broad heterotic group were then dated by year of release, any inbred line without a year of release affiliated with it was discarded from further analysis.

Our second approach effectively scans for  For further information - see \kc{Berg and Coop 2014}

Our final approach used a more recent version of \kc{Nielsen} of finding the signature of positive selection using SWeeD

\subsection*{Data Availability}

Data for GBS 2.7 are available on PANZEA.\jri{add linke} All scripts are available on github (give directory).

%At the end of the Materials and Methods section, include a statement on reagent and data availability. Please read the Data and Reagent Policy before writing the statement. Make sure to list the accession numbers or DOIs of any data you have placed in public repositories. List the file names an descriptions of any data you will upload as supplemental information. The statement should also include any applicable IRB numbers. You may include specifications for how to properly acknowledge or cite the data.

%For example: Strains are available upon request. File S1 contains detailed descriptions of all supplemental files. File S2 contains SNP ID numbers and locations. File S3 contains genotypes for each individual. Sequence data are available at GenBank and the accession numbers are listed in File S3. Gene expression data are available at GEO with the accession number: GDS1234. Code used to generate the simulated data is provided in file S4. 


\section*{Results and Discussion}

\subsection{Identification of heterotic groups}

\begin{figure}[htbp]
\centering
\includegraphics[width=\linewidth]{6_clusters_placeholder.png}
\caption{Six genetic clusters representing the broad heterotic groups of modern maize from the Ames panel using GBS 2.7 aligned to version 3 of the B73 genome 
}
\label{fig:6clusters}
\end{figure}



\section*{Acknowledgements}
Financial support for this work came from NSF grant IOS-1238014, USDA grant 2009-65300-05668, and DuPont Pioneer.  We'd like to thank DuPont Pioneer as well for their kind support gathering information about years of release. We thank Graham Coop, James Holland, Edward Buckler, \X and \X reviewers for helpful discussion. 

\bibliography{example-bibliography}

\end{document}